components/Icons.es6.js

```diff
@@ -1845,6 +1845,16 @@ function Zap ({ className, width='11', height='16', color="#D7D7DE", viewBox="0
   )
 }
 
+function Webpage ({className, width="16", height="16", color = "#84839C"}) {
```

Comment:
Wanna add this to `src/react-demo/Icons.es6.js` as well. Reference: https://github.com/b12io/metronome/pull/38/files